flexmeasures/ui/templates/crud/asset_properties.html

```diff
@@ -0,0 +1,335 @@
+                                    <div class="col-md-6">
+                                        <input class="form-control" id="asset-id" name="asset-id" type="text"
+                                            value="{{ asset.id }}" disabled></input>
+                                    </div>
+                                </div>
+
+                                <div class="form-group">
+                                    {{ asset_form.attributes.label(class="col-sm-3 control-label") }}
+                                    <div class="col-md-3">
+                                        {{ asset_form.attributes(class_="form-control") }}
+                                        {% for error in asset_form.errors.attributes %}
+                                        <span style="color: red;">[{{error}}]</span>
+                                        {% endfor %}
+                                    </div>
+                                </div>
+
+                                <div class="form-group">
+                                    <label for="flex_context" class="col-sm-6 control-label">Flex Context: </label>
+
+                                    <button class="btn btn-primary" type="button" data-bs-toggle="modal"
+                                        data-bs-target="#flexContextModal">
+                                        Edit
+                                    </button>
+                                </div>
+
+                                <div class="form-group">
+                                    <label for="sensors_to_show" class="col-sm-6 control-label">Sensor Graphs: </label>
+
+                                    <button class="btn btn-primary" type="button" data-bs-toggle="modal"
+                                        data-bs-target="#sensorsToShowModal">
+                                        Edit Graphs
+                                    </button>
+                                </div>
+
+                                <div class="form-group">
+                                    <label class="control-label">Location</label>
+                                    <small>(Click map to edit latitude and longitude in form)</small>
+                                    <div id="mapid"></div>
+                                    <button class="btn btn-sm btn-responsive btn-success create-button" type="submit"
+                                        value="Save"
+                                        style="margin-top: 20px; float: right; border: 1px solid var(--light-gray);">
+                                        Save
+                                    </button>
+                                </div>
+                            </div>
+                        </fieldset>
+                    </form>
+                </div>
+            </div>
+            {% endif %}
+   
+        </div>
+   <div class="col-md-8">
+     
+
```

Comment:
Actually, here it would be great to show the properties that are in the edit form on the left, just in a normal info table, above the other tables for sensors and child assets.

Should be easy as all this information is already loaded for the edit form.